evalml/automl/automl_search.py

```diff
@@ -585,6 +586,17 @@ def _add_baseline_pipelines(self, X, y):
 
         return False
 
+    @staticmethod
+    def _get_mean_cv_scores_for_all_objectives(cv_data):
+        scores = defaultdict(int)
+        objective_names = set([name.lower() for name in get_all_objective_names()])
+        n_folds = len(cv_data)
+        for fold_data in cv_data:
+            for field, value in fold_data['all_objective_scores'].items():
+                if field.lower() in objective_names:
+                    scores[field] += value
+        return {objective_name: score / n_folds for objective_name, score in scores.items()}
```

Comment:
Cool. If `score` is `int` this won't carry the remainder, so for safety please do `float(score) / n_folds`